docs/src/content/pages/multi-select-field.mdoc

```diff
@@ -1,6 +1,29 @@
 ---
 title: Multi-Select field
 ---
+The `multiselect` field is similar to the `select` field but allows you to select zero, one or many options.
+
+Checkboxes will be used in the Admin UI for this field.
```

Comment:
I haven't checked the UI, but my assumption would be that it is a select list with multiselect?
```suggestion
A select list with checkboxes will be used in the Admin UI for this field.
```

Reply:
Hmmmm no it's literally a set of checkboxes, see screenshot.

<img width="196" alt="CleanShot 2023-05-30 at 13 21 39@2x" src="https://github.com/Thinkmill/keystatic/assets/485747/d5489cad-2faa-4714-8e21-0a4916e5f655">

Reply:
🤦 really should have checked the UI first. My bad!